website/src/_posts/2019-04-liftoff-08.md

```diff
@@ -0,0 +1,53 @@
+---
+title: "Day 8"
+date: 2019-04-02
+author: samuel
+image: "https://uppy.io/images/blog/30daystoliftoff/day08.jpg"
+series: 30 Days to Liftoff
+seriesSuffix: 'of 30'
+---
+
+Welcome to Day 8 of our ongoing thirty-day blog post challenge toward the **Uppy 1.0 release on April 25**! Yesterday was the 1st of April, the day we really kickstarted our coding speedrun towards 1.0
+
+The entire team during the weekly call yesterday iterated on the whole process again, moving a lot of cards from the `todo` lane to the `in progress` lane. This is what our board currently looks like.
+
+<center><br /><img width="400" src="/images/blog/30daystoliftoff/2019-04-02-asana-board.png"><br /></center>
+
+Let's dive into the updates we have ready to share with you.
+
+
+<!--more-->
+
+<center><br /><img width="400" src="/images/blog/30daystoliftoff/day07.jpg"><br /></center>
+
+## Done
+- [Artur](https://transloadit.com/about/#artur) merged this [PR](https://github.com/transloadit/uppy/pull/1393) which removed the use of `window.location` from companion's client coutrsey of [Ife](https://transloadit.com/about/#ife)
+- [Alex](https://transloadit.com/about/#alex) improved the design of the drag to upload stage. <br />
+Before:<br />
+<center><img width="400" src="/images/blog/30daystoliftoff/2019-04-02-before.png"></center>
+After:<br />
+<center><img width="400" src="/images/blog/30daystoliftoff/2019-04-02-after.png"></center>
+- [Evgenia](https://github.com/lakesare) made list items even out using the `justify-content: space-between` CSS property.
+
```

Comment:
that’s incorrect, let’s just say she “made file items even out properly”, she didn’t use `space-between`.

Reply:
Ok thanks, I'll fix it